docs/content/en/blog/control-plane-local-docker-compose.md

```diff
@@ -19,8 +19,7 @@ This blog is for those who would like to:
 The general architecture of PipeCD Control Plane is as below.
 ![](/images/control-plane-components.png)
 
-> Note: See [Architecture Overview](docs/user-guide/managing-controlplane/architecture-overview/) doc for details.
-
+> Note: See [Architecture Overview](/docs/user-guide/managing-controlplane/architecture-overview/) doc for details.
```

Comment:
It seems not to work on `make run/site`👀

<img width="748" alt="404_Page_not_found___PipeCD" src="https://github.com/pipe-cd/pipecd/assets/40124947/a46180ba-3b3d-4318-837a-e4c4b77bdd72">

How about using `/docs-dev` instead of `/docs` ?

Reply:
That might be because `make run/site` does not show `docs` (why??)

![image](https://github.com/pipe-cd/pipecd/assets/97105818/2d257ded-45e5-4759-9bed-49177c20073c)


Please visit https://pipecd.dev/blog/2024/03/14/control-plane-on-local-by-docker-compose/ and modify the `href`. That worked.

![image](https://github.com/pipe-cd/pipecd/assets/97105818/72d593be-6a25-4fe3-b9df-a4025dc95e1b)

Reply:
@t-kikuc Got it. You're right. `make run/site` couldn't redirect the link. 

It seems that the site is hosted by our file server, which works in a container. `/docs` is redirected to the latest version `/docs-<latest-version>`.
https://github.com/pipe-cd/pipecd/blob/b564c82c2b2321c3f4112365149ba606677285a5/docs/main.go#L28-L49

But `make run/site` uses `hugo server` command, which creates the server internally to check the behavior and doesn't use our server. 😇

So I tried to execute `hugo` to build the files and `go run main.go` in the docs dir, then it worked well.
(I added the small fix to check the build result, change from `"/public"` to `"./public"` at this [line](https://github.com/pipe-cd/pipecd/blob/b564c82c2b2321c3f4112365149ba606677285a5/docs/main.go#L28-L29))